libs/execution/terragrunt.go

```diff
@@ -104,9 +104,9 @@ func (terragrunt Terragrunt) runTerragruntCommand(command string, printOutputToS
 	env := os.Environ()
 	env = append(env, "TF_CLI_ARGS=-no-color")
 	env = append(env, "TF_IN_AUTOMATION=true")
-	env = append(env, "TERRAGRUNT_NO_COLOR=true")
-	env = append(env, "TERRAGRUNT_NON_INTERACTIVE=true")
-	env = append(env, "TERRAGRUNT_FORWARD_TF_STDOUT=1")
+	env = append(env, "TG_NO_COLOR=true")
```

Comment:
we need to specify both TG_xxx and TERRAGRUNT_xxx to be backward compatible with older terragrunt versions which had TERRAGRUNT_xxx